skillsync/src/Pages/Contact/Contact.js

```diff
@@ -8,16 +8,24 @@ import phone from "../../assets/phone.jpg";
 import emailjs from "@emailjs/browser";
 
 export const Contact = () => {
-  const form = useRef();
+  const nameRef = useRef();
+  const emailRef = useRef();
+  const messageRef = useRef();
 
   const sendEmail = (e) => {
     e.preventDefault();
 
-    emailjs
-      .sendForm(
+  const templateParams = {
+    user_name: nameRef.current.value,
+    user_email: emailRef.current.value,
+    message: messageRef.current.value,
+   };
+
+   emailjs
+      .send(
         "service_vyb0ww4",
         "template_lp1roh3",
-        form.current,
+        templateParams,
         "lXtWrN83VcO2t54Is"
```

Comment:
Is this an API key? @Lakshi99

Reply:
No aiya, It is an object that holds the dynamic data(user name, user email and message) that passes to the EmailJS template.